DigiMeSDK/Core/Classes/DMEPushClient.m

```diff
@@ -32,6 +32,15 @@ - (instancetype)initWithConfiguration:(DMEClientConfiguration *)configuration
     return self;
 }
 
+- (void)openDMEAppForPostboxImport
+{
+    if ([self.appCommunicator canOpenDMEApp])
+    {
+        DMEOpenAction *action = @"postbox/import";
+        [self.appCommunicator openDigiMeAppWithAction:action parameters: [[NSDictionary alloc] init]];
```

Comment:
Can use the literal syntax here (and remove the space after colon):
`... parameters:@{}];